src/test/java/net/masterthought/cucumber/generators/TagsOverviewPageTest.java

```diff
@@ -95,14 +96,15 @@ public void generateTagValues_ReturnsTagValues() {
         List<TagObject> allTags = this.tags;
 
         // when
-        List<String> labels = TagsOverviewPage.generateTagValues(allTags);
+        String[][] labels = TagsOverviewPage.generateTagValues(allTags);
 
         // then
-        assertThat(labels).containsExactly(
-                "[62.50, 100.00, 100.00]",
-                "[6.25, 0.00, 0.00]",
-                "[12.50, 0.00, 0.00]",
-                "[6.25, 0.00, 0.00]",
-                "[12.50, 0.00, 0.00]");
+        assertThat(labels).isEqualTo(new String[][]{
```

Comment:
I believe there is some containsExactly for two dimentional arrays - take a look if it won't be simpler